administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -1,99 +1,73 @@
 import { Colors, FormGroup } from '@blueprintjs/core'
-import { PartialMessage } from '@bufbuild/protobuf'
 import { TextField } from '@mui/material'
-import { ReactElement } from 'react'
+import React, { ReactElement } from 'react'
 
-import { CardExtensions } from '../../generated/card_pb'
+import useWindowDimensions from '../../hooks/useWindowDimensions'
 import PlainDate from '../../util/PlainDate'
-import { Extension } from './extensions'
+import { Extension, ExtensionComponentProps } from './extensions'
 
-type BirthdayState = { birthday: number }
+export const BIRTHDAY_EXTENSION_NAME = 'birthday'
+export type BirthdayExtensionState = { [BIRTHDAY_EXTENSION_NAME]: number }
 
 const minBirthday = new PlainDate(1900, 1, 1)
+const getInitialState = (): BirthdayExtensionState => ({ birthday: minBirthday.toDaysSinceEpoch() })
 
-class BirthdayExtension extends Extension<BirthdayState, null> {
-  public readonly name = BirthdayExtension.name
+const BirthdayForm = ({ value, setValue, isValid }: ExtensionComponentProps<BirthdayExtensionState>): ReactElement => {
+  const { viewportSmall } = useWindowDimensions()
+  const inputColor = value === getInitialState() ? Colors.GRAY1 : Colors.BLACK
```

Comment:
Shouldn't we rather set `null` as initial state and render `01.01.1900` as placeholder?

Reply:
we still have design issues for that

Reply:
the problem is that then the current date is used if you open the datepicker. we can do that but when i remember right the more annoying thing was to the scroll hell if not setting a default value to sth closer to most common birthdays